common/build.gradle

```diff
@@ -19,7 +19,7 @@ dependencies {
     testImplementation group: 'org.mockito', name: 'mockito-core', version: '4.4.0'
 
     compileOnly group: 'org.apache.commons', name: 'commons-text', version: '1.10.0'
-    compileOnly group: 'com.google.code.gson', name: 'gson', version: '2.10.1'
+    implementation group: 'com.google.code.gson', name: 'gson', version: '2.10.1'
```

Comment:
Why have to change this part? Don't suggest change common package if it doesn't break your code.

Reply:
Tested with neural-search plugin, if don't change this, there'll be `java. lang. NoClassDefFoundError: com/google/gson/Gson`, changing this can fix this issue.